packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts

```diff
@@ -52,8 +68,18 @@ export interface FaceLivenessDetectorConfig {
    * default is https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file
    */
   faceModelUrl?: string;
+
+  /**
+   * Optional parameter providing your own credentials when calling the Rekognition stream endpoint
```

Comment:
Nit avoid second person references in documentation
```suggestion
   * Optional parameter allowing usage of custom credential providers when calling the Rekognition stream endpoint
```